metaflow/datastore/datastore_backend.py

```diff
@@ -226,12 +227,7 @@ def load_bytes(self, paths):
         -------
         CloseAfterUse :
             A CloseAfterUse which should be used in a with statement. The data
-            in the CloseAfterUse will be a dictionary string -> (path, dict).
-            The key is the path fetched and the value is a tuple containing:
-              - a path indicating the file that needs to be read to get the object.
-                This path may not be valid outside of the CloseAfterUse scope
-              - a dictionary containing any additional metadata that was stored
-              or None if no metadata was provided.
-            If the path could not be loaded, returns None for that path
+            in the CloseAfterUse will be an iterator over (key, path, metadata)
+            tuples. Path and metadata will be None if the key was missing.
```

Comment:
I would revert to the old comment slightly modified. The iterator is over key, path, metadata and path and metadata are None if the key is missing but metadata can also be none even if the key is not missing (which is unclear from this comment).